code/modules/clothing/suits/wintercoats.dm

```diff
@@ -29,6 +31,42 @@
 		/obj/item/toy,
 	)
 
+	generate_hood_overlay()
+
+
+/obj/item/clothing/suit/hooded/wintercoat/examine(mob/user)
+	. = ..()
+
+	. += span_notice("<b>Alt-click</b> to [zipped ? "un" : ""]zip.")
+
+
+/obj/item/clothing/suit/hooded/wintercoat/AltClick(mob/user)
+	. = ..()
+
+	if (. == FALSE) // Direct check for FALSE, because that's the specific case we want to propagate, not just null.
+		return FALSE
+
+	zipped = !zipped
+	worn_icon_state = "[initial(icon_state)][zipped ? "_t" : ""]"
+	balloon_alert(user, "[zipped ? "" : "un"]zipped [src]")
```

Comment:
```suggestion
	balloon_alert(user, "[zipped ? "" : "un"]zipped")
```

Really easy for a super-long clothing name to make a really big and ugly balloon alert, this is plenty imo

Reply:
True, I don't know why I left that there lol